### ISSUE TITLE
[Suggestion] Custom logger config

### ISSUE THREAD

**Issue by Juuxel** [FabricMC/fabric#400]
Minecraft's default logger config hides the logger name. A custom logger config that shows them would be useful for debugging to see the class/mod where the message comes from, and modders wouldn't have to manually add mod names to log messages.

**Comment by i509VCB** [comment]
I think this was covered by Loom in 0.2.7.

**Comment by modmuss50** [comment]
Yes, but the next stage is to get it out of loom and into production. This is easier said than done however.

**Comment by maityyy** [comment]
still actual?
